Sources/TokenGeneration/Extensions/Data+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation.NSData
+
+extension Data {
+    /// Base 64 encoding the string.
+    ///
+    /// Removing unallowed URL encoded character:
+    ///   - "="
+    ///
+    /// Replacing unallowed URL encoded characters:
+    ///   - "+" with "-"
+    ///   - "/" with "_"
+    var base64URLEncodedString: String {
```

Comment:
Do we have a link to any specifications that detail these constraints incase anyone wants further reading?

Reply:
Want to link that in the comment?

Reply:
I think `unallowed` should be `disallowed` or `unsupported`?